### PR TITLE
Leaflet: Add parameter to disable mouse scrolling (fix #316)

### LINKED ISSUE #316: can't disable mouse scrolling: Scrollwheel zooms map in and out when scrolling through the page
### Setup
https://wiki.sha2017.org/w/Special:Version

### Issue

Scrolling through a page using the mouse zooms the map when hovering over it. Would like to disable mouse scrolling and just use the control.

Example:
https://www.semantic-mediawiki.org/wiki/Maps_examples/OpenLayers_basic_map

Go here and and use your mouse wheel to scroll through the page.

Additionally: the static option doesn't work to fix this: when there are markers with large text-balloons, the position of the map is changed to show the text balloon after clicking: resulting you cannot use the mouse or anything else to show the original map except reloading the page. This makes the static option unusable.

### PR DESCRIPTION
**Pull request by hidrarga**
Should fix #316

### CODE REVIEW

File: includes/Maps_Mapper.php [hunk 1]
```diff
@@ -193,6 +193,11 @@ private static function getEvenMawrCommonParameters() {
 			'default' => false,
 		];
 
+		$params['scrollwheelzoom'] = [
+			'type' => 'boolean',
+			'default' => true,
+		];
```
Comment (JeroenDeDauw) [inline includes/Maps_Mapper.php]: Please add this to the leaflet specific parameters if it is only supported for that mapping service

File: i18n/en.json [hunk 2]
```diff
@@ -50,6 +50,7 @@
 	"maps-coordinates-par-location": "The coordinates you want to format.",
 	"maps-coordinates-par-format": "The target format for the coordinates.",
 	"maps-coordinates-par-directional": "Indicates if the coordinates should be outputted directional or not.",
+	"maps-coordinates-par-scrollwheelzoom": "Indicates if mouse scrolling should be enabled or not.",
```
Comment (hidrarga) [inline i18n/en.json]: Let me know if that sentence is good enough ;)

File: includes/services/Leaflet/Maps_Leaflet.php [hunk 3]
```diff
@@ -67,6 +67,11 @@ public function addParameterInfo( array &$params ) {
 			'message' => 'maps-par-enable-fullscreen',
 		];
 
+		$params['scrollwheelzoom'] = [
+			'type' => 'boolean',
+			'default' => true,
+		];
```
Comment (JeroenDeDauw) [inline includes/services/Leaflet/Maps_Leaflet.php]: 'message' => 'maps-par-scrollwheelzoom',

File: i18n/en.json (hunk 2 above)
Comment (JeroenDeDauw) [reply i18n/en.json]: Yeah but the key is now

'maps-par-scrollwheelzoom'

This is not a parameter for the coordinates parser function :)

File: includes/services/Leaflet/Maps_Leaflet.php (hunk 3 above)
Comment (hidrarga) [reply includes/services/Leaflet/Maps_Leaflet.php]: Oops, sorry! Didn't pay enough attention here..

**Comment by JeroenDeDauw** [comment]
Thanks! Feel free to also add a line to the release notes

**Comment by PeterTheOne** [comment]
qqq.json change still seems to be missing..

**Comment by kghbln** [comment]
@PeterTheOne Will add this via twn.

**Comment by kghbln** [comment]
https://translatewiki.net/wiki/MediaWiki:Maps-par-scrollwheelzoom/qqq

**Pull request merged**